topic_spec.json

```diff
@@ -0,0 +1,24 @@
+{
+    "topics" : [
+        {"name": "Topic1",
+         "interval_ms": "100",
+         "inject_timestamp" : true,
+         "QoS": 1,
+         "payload_encoding": "eterm",
+         "payload": {"foo1" : "bar1", "timestamp": "0", "Data": "10"},
+         "stream" : 0,
```

Comment:
stream 0 is default, signal stream solution used by TCP TCP/TLS and QUIC single stream.
stream >0 is for multi stream features.